modules/test/poshi-runner/src/main/java/com/liferay/poshi/runner/util/StringUtil.java

```diff
@@ -533,6 +533,20 @@ public static String stripBetween(String s, String begin, String end) {
 		return sb.toString();
 	}
 
+	public static byte[] toByteArray(String text) {
```

Comment:
@lesliewong92 Is the parameter, "text" a comma delimited list of numeric values? I'm not sure how this is going to be used.